.ado/templates/setup-repo.yml

```diff
@@ -1,9 +1,9 @@
 # This template install dependencies, builds, and bundles this repo
 steps:
-  - task: NodeTool@0
-    inputs:
-      versionSource: 'fromFile'
-      versionFilePath: '.node-version'
+  # - task: NodeTool@0
```

Comment:
should this stay commented?

Reply:
The windows build doesn't work if we use NodeTool to install node (even though its already got node 18?  🤷.  -- And all the builds are working currently.  I can remove the code completely as a follow up PR.